### PR TITLE
docs(all): improve the Angular FAQ entry

### PR DESCRIPTION
**Pull request by clebert**

### CODE REVIEW

File: docs/help/faq.md [hunk 1]
```diff
@@ -30,16 +30,19 @@ App][feature-app-in-feature-app].
...
-App][writing-a-feature-app]. The [`@feature-hub/react`][react-api] package could
-be used to integrate the application as a [DOM Feature App][dom-feature-app].
-However, if you don't want to integrate other React Feature Apps, you probably
-should build a Feature Hub Integrator without React.
-
-[own-feature-service-definitions]:
-  /docs/guides/writing-a-feature-app#ownfeatureservicedefinitions
+App][writing-a-feature-app]. The [`@feature-hub/dom`][dom-api] or the
+[`@feature-hub/react`][react-api] package can be used to integrate the
+application as a DOM Feature App. We have created a [demo][angular-feature-app]
```
Comment (fahrradflucht) [inline docs/help/faq.md]: Do we want to write as `we` here or, do we just want to say that "There is a demo..."?

Comment (clebert) [reply docs/help/faq.md]: Yes you're right

Comment (clebert) [reply docs/help/faq.md]: done

File: docs/help/faq.md [hunk 2]
```diff
@@ -30,16 +30,19 @@ App][feature-app-in-feature-app].
...
-be used to integrate the application as a [DOM Feature App][dom-feature-app].
-However, if you don't want to integrate other React Feature Apps, you probably
-should build a Feature Hub Integrator without React.
-
-[own-feature-service-definitions]:
-  /docs/guides/writing-a-feature-app#ownfeatureservicedefinitions
+App][writing-a-feature-app]. The [`@feature-hub/dom`][dom-api] or the
+[`@feature-hub/react`][react-api] package can be used to integrate the
+application as a DOM Feature App. There is a [demo][angular-feature-app] that
+shows the integration of an Angular application into an `@feature-hub/dom`
```
Comment (unstubbable) [inline docs/help/faq.md]:
```suggestion
shows the integration of an Angular application into a `@feature-hub/dom`
```

**Pull request merged**